http/context_test.go

```diff
@@ -29,6 +29,13 @@ func (s *ContextTestSuite) TestWithValue() {
 	s.Equal("world", s.ctx.Value("Hello"))
 }
 
+func (s *ContextTestSuite) TestWithValueUsingCustomKey() {
+	var myKey struct{}
+	s.ctx.WithValue(myKey, "hola")
+	s.Equal("hola", s.ctx.Value(myKey))
+	s.NotEqual("hola", s.ctx.Value("myKey"))
+}
+
```

Comment:
`TestWithValue` and `TestWithValueUsingCustomKey` can be merged.

Reply:
done in https://github.com/goravel/framework/pull/630/commits/df493e734a961e18b74e2fd2796ed1100a2bd46a